planner/core/optimizer.go

```diff
@@ -414,6 +414,8 @@ func enableParallelApply(sctx sessionctx.Context, plan PhysicalPlan) PhysicalPla
 		supportClone := err == nil // limitation 2
 		if noOrder && supportClone {
 			apply.Concurrency = sctx.GetSessionVars().ExecutorConcurrency
+		} else {
+			sctx.GetSessionVars().StmtCtx.AppendWarning(errors.Errorf("Some apply executors can not be executed parallel"))
```

Comment:
`Some apply operators can not be executed in parallel`

Reply:
Thanks for your review!